### DISCUSSION TITLE
Create group in Discord

### DISCUSSION THREAD

**Discussion by II11II** [serverpod/serverpod#222] [Ideas]
It would be fantastic to gather those who are interested in ServerPod and help them to understand using this framework. I believe it helps to grow the community of this project🙂

**Comment by vlidholt** [comment]
We are planning to do this in the future when the community starts to become a little bit larger.

**Reply by ajay-k07** [reply]
I think this will be a perfect time to do that.so that we can collaborate easily and let's get prepared for version 1.0 😌

**Comment by AndryHTC** [comment]
In the meantime, any idea for a better suited platform than Discord?

**Reply by vlidholt** [reply]
How about a platform built in Serverpod? The chat module supports, attachments, and is pretty full fledged. It could be part of the Serverpod Insights app (or an external web app), depending on what people prefer.

**Reply by ajay-k07** [reply]
That's really a great idea. I'm ready to work on this application. Please let me know how we can further proceed

**Reply by AndryHTC** [reply]
It depends...

Could be a great idea if we value the community platform less than showcasing Serverpod.

But, as creating a community platform it's not the core business of Serverpod, using something else may be more appropriated.

I would like to find a platform that gives everything out of the box (improvements included), letting us to contribute to Serverpod without spending time to maintaining another platform.

Discord is not bad but it's more gaming oriented. Slack is not bad too but... maybe there is something as cool as Serverpod we can use for the community 😄

**Reply by vlidholt** [reply]
Generally, I think Github isn't too bad. It lack the "directness" of Slack and Discord, but it's nice to be able to reference issues and I like that everything is in one place.

**Reply by ajay-k07** [reply]
GitHub is good but we are not able to interact much as we can do on slack or discord

**Reply by AndryHTC** [reply]
I concur with @vlidholt that it might be beneficial to utilize other platforms for real-time live communication, while reserving asynchronous replies for more thoughtful and well-reasoned responses that can be of value to the community. As developers, we often find that asynchronous communication allows for more in-depth and considered discussions.

**Reply by yahu1031** [reply]
> How about a platform built in Serverpod? The chat module supports, attachments, and is pretty full fledged. It could be part of the Serverpod Insights app (or an external web app), depending on what people prefer.

I knew this was coming. Haha, I was about to say. 🤣

**Reply by tusharbhambere** [reply]
> GitHub is good but we are not able to interact much as we can do on slack or discord

Yes but I just bookmarked Serverpod GitHub Repo😵

**Reply by AndryHTC** [reply]
> Generally, I think Github isn't too bad. It lack the "directness" of Slack and Discord, but it's nice to be able to reference issues and I like that everything is in one place. 

So we can just add a little bit of real-time features to GitHub

Here we go: https://gitter.im/
